include/highfive/bits/H5Reference_misc.hpp

```diff
@@ -64,6 +64,33 @@ inline Object Reference::get_ref(const Object& location) const {
     return Object(res);
 }
 
+namespace details {
+template <>
+struct inspector<Reference> {
```

Comment:
maybe not a bad idea to move this to the other `inspector` template classes.

Reply:
There is a problem of includes. And I think it's better, at the end, there will be the `H5::Reference` class only declared and used here.